scripts/jira/cloner.py

```diff
@@ -277,48 +317,48 @@ def query_build(issue, user):
 
     args = parser.parse_args()
 
-    connection = jira.JIRA(
+    conn = jira.JIRA(
         server=JIRA_SERVER,
         token_auth=args.token)
 
     jql_query = query_build(args.issue, args.user)
     print(f"JQL Query: '{jql_query}'")
 
     try:
-        query = connection.search_issues(jql_str=jql_query, maxResults=9999)
+        query = conn.search_issues(jql_str=jql_query, maxResults=9999)
         print(f"Scanning {len(query)} issues")
     except Exception as e:
         print(f"Unable to retrieve issues: {e}")
-        exit(1)
+        sys.exit(1)
 
-    actions = []
+    actions_list = []
     for i in tqdm(range(len(query))):
         try:
-            issue = query[i]
+            issuee = query[i]
         except Exception as e:
             print(f"Unable to retrieve issue {query[i]}: {e}")
             continue
-        actions.extend(scan_issue(issue, connection))
+        actions_list.extend(scan_issue(issuee, conn))
 
-    print(tabulate([[x.issue, x.comment, 'Y' if x.action is None else 'N'] for x in actions], headers=['Issue', 'Action', 'Manual']))
+    print(tabulate([[x.issue, x.comment, 'Y' if x.action is None else 'N'] for x in actions_list], headers=['Issue', 'Action', 'Manual']))
     print()
 
-    actions = list(filter(lambda x: x.action is not None, actions))
-    if len(actions) == 0:
+    actions_list = list(filter(lambda x: x.action is not None, actions_list))
+    if len(actions_list) == 0:
         print("No automatic actions to perform.")
-        exit(0)
+        sys.exit(0)
 
     answer = ''
     if args.auto_accept:
         answer = 'y'
     while answer not in ['y', 'n']:
-        answer = input(f'Perform {len(actions)} non manual actions? [Y/N]').lower()
+        answer = input(f'Perform {len(actions_list)} non manual actions? [Y/N]').lower()
     if answer == 'n':
```

Comment:
If  I'm not wrong, `answer` is not a constant here, it's a variable. Still, pylint is thinking it as a constant and suggesting to use UPPER_CASE.
`C0103: Constant name "answer" doesn't conform to UPPER_CASE naming style (invalid-name)`

Reply:
I wonder if line 311 is making it think it's a constant? Maybe that's another rule we want to turn off?

Reply:
It's complaining for both 311 and 313 lines.  And [C0103](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/invalid-name.html) is a very broad check for all types of naming convention, so if we want to turn it off, we can do that for only these two lines. Like this:


```python
    answer = '' #pylint: disable=invalid-name
    if args.auto_accept:
        answer = 'y' #pylint: disable=invalid-name
    while answer not in ['y', 'n']:
        answer = input(f'Perform {len(actions_list)} non manual actions? [Y/N]').lower()
    if answer == 'n':
        sys.exit(0)

```

Reply:
Added `#pylint: disable=invalid-name` for these two line to turn off the rule. It seems like a false positive.